AnalogButtons.cpp

```diff
@@ -27,7 +27,7 @@ void AnalogButtons::check() {
 	// In case this function gets called very frequently avoid sampling the analog pin too often: max frequency is 50Hz
 	if (millis() - time > 120) {
 		time = millis();
-		uint16_t reading = analogRead(pin);
```

Comment:
reading can **never** be negative, this doesn't make any sense, not even within the context of #5